modules.json

```diff
@@ -7,7 +7,7 @@
                 "nf-core": {
                     "cat/fastq": {
                         "branch": "master",
-                        "git_sha": "c8e35eb2055c099720a75538d1b8adb3fb5a464c",
+                        "git_sha": "76cc4938c1f6ea5c7d83fed1eeffc146787f9543",
                         "installed_by": ["modules"]
```

Comment:
Dont the `samplesheet_check`,  `dumpsoftwareversions` and `multiqc` have been also updated?

Reply:
Highly possible, let me see what I can do about it